lib/requester/core.js

```diff
@@ -275,11 +277,25 @@ module.exports = {
             requestMethod = (typeof request.method === 'string') ? request.method.toLowerCase() : undefined,
             bodyIsEmpty = requestBody.isEmpty(),
             bodyIsDisabled = requestBody.disabled,
-            bodyContent = requestBody[requestBodyType];
+            bodyContent = requestBody[requestBodyType],
+            protocolProfileBehavior = requestBody.protocolProfileBehavior,
```

Comment:
Add a comment explaining that this initialization is needed separately to allow checking for both: requests with as well as without this property.